lib/active_admin/view_helpers/download_format_links_helper.rb

```diff
@@ -2,6 +2,18 @@ module ActiveAdmin
   module ViewHelpers
     module DownloadFormatLinksHelper
 
+      def build_download_formats(download_links)
+        download_links = instance_exec(&download_links) if download_links.is_a?(Proc)
+
+        if download_links.is_a?(Array) && !download_links.empty?
```

Comment:
Why do we need to support passing a block here, by the way? The [case we are testing](https://github.com/activeadmin/activeadmin/pull/4867/files#diff-21130d3a612b3d5292e9e08016a4e5e6R81) is a bit cumbersome and not dynamic.